ml-agents/mlagents/trainers/ppo/optimizer_torch.py

```diff
@@ -187,7 +187,7 @@ def update(self, batch: AgentBuffer, num_sequences: int) -> Dict[str, float]:
 
         self.optimizer.step()
         update_stats = {
-            "Losses/Policy Loss": policy_loss.item(),
+            "Losses/Policy Loss": torch.abs(policy_loss).item(),
```

Comment:
Why is this needed ?

Reply:
> Does this line make sense any more https://github.com/Unity-Technologies/ml-agents/blob/master/ml-agents/mlagents/trainers/ppo/optimizer_torch.py#L180?

The masked_mean was actually _not_ being used properly before (it was just a single value). It is now, as the entropy is BxN.

Reply:
I do not see why that helps much. The loss is roughly `mean(Prob x Advantage)`. This loss can fluctuate above/bellow zero. What we report is then `mean(abs(mean(Prob x Advantage)))`  (Because we do the mean of the aggregated stats) which is just very weird. Maybe we can report : 
 - `mean(mean(abs(Prob x Advantage)))` (Not sure if it makes sense)
 - `mean(mean(Prob x Advantage))` (remove the abs, is what it says it is)
 - `mean(std_dev(Prob x Advantage))` If the fluctuations are large, the std_dev should tell us
 - `mean(max(abs(Prob x Advantage)))` If we want to know what is the largest error we have

Reply:
TBH none of these are really interpretable from the perspective of what we'd normally expect from "loss" (i.e. loss goes down over time). Seems like the last one might get the right behavior, but it's a real stretch to call it a "Policy Loss". Maybe we can go that route and call it "Policy Error" or something similar. 

Otherwise, just going with `mean(mean(Prob x Advantage))` is the most correct, but maybe not useful.

Reply:
I've been thinking more about this - I think we should leave it with the `abs` at least until the first release after PyTorch default switch, so that users who are comparing TF and Torch will have the exact same plots with no surprises.

Reply:
Ok, but we should revisit once TF is deprecated.

Reply:
100% for sure. I think we probably want both the `std_dev` and `mean` at the end of the day.